client/my-sites/plan-features/style.scss

```diff
@@ -1,3 +1,29 @@
+.plan-features {
+	margin: 0 0 16px;
+	border: solid 1px lighten( $gray, 27% );
+	background-color: lighten( $gray, 35% );
+
+	&.is-popular {
+		margin-top: 35px; // popular banner + margin
```

Comment:
Maybe add some local sass variables, so this will be updated automatically when values height or margin change.